azure-pipelines.yml

```diff
@@ -1,12 +1,20 @@
 trigger:
-  batch: true
```

Comment:
So we don't want batching anymore because of different competing build definitions?

Reply:
I don't think we want batching in the runtime repository and it makes diagnosing issues in this repository difficult at times.
Since core-setup doesn't run a public CI job, the only post-merge validation is a batched official build. When that breaks, you have to figure out which commit caused the break, which takes a significant amount of time. Since we don't run tests on limited-hardware configurations anyway, I don't think the minor increase in number of official builds will cause any issues.

Reply:
ok, thanks.